tests/migrations/test_writer.py

```diff
@@ -271,29 +271,41 @@ class IntEnum(enum.IntEnum):
 
         self.assertSerializedResultEqual(
             TextEnum.A,
-            ("migrations.test_writer.TextEnum['A']", {'import migrations.test_writer'})
+            (
+                "migrations.test_writer.WriterTests.test_serialize_enums.<locals>.TextEnum['A']",
+                {'import migrations.test_writer'}
```

Comment:
I would move all enums to the top of this file (in a separate commit) to get rid of `<locale>`, we should also add one `Enum` to the `WriterTests` to test the fix, e.g.
```python
class WriterTests(SimpleTestCase):
    """
    Tests the migration writer (makes migration files from Migration instances)
    """
    class InnerEnum(enum.IntEnum):
        A = 1
        B = 2

    ...

    def test_serialize_enums(self):
        ...
        self.assertSerializedResultEqual(
            self.InnerEnum.A,
            (
                "migrations.test_writer.WriterTests.InnerEnum['A']",
                {'import migrations.test_writer'}
            )                          
        )
        self.assertSerializedEqual(self.InnerEnum.A)
```